addon/components/o-s-s/panel/row.hbs

```diff
@@ -1,10 +1,10 @@
-<div class="oss-panel-content--row" ...attributes>
+<div class="oss-panel-content--row {{if @disabled 'oss-panel-content--row-disabled'}}" ...attributes>
```

Comment:
It might be a good time to add a stories file for this one

Reply:
![image](https://github.com/user-attachments/assets/09e9dc58-90f0-4960-95ee-040653da2c94)